lib/realtime/stop_time_updates_by_trip.ex

```diff
@@ -0,0 +1,16 @@
+defmodule Realtime.StopTimeUpdatesByTrip do
+  alias Concentrate.StopTimeUpdate
+  alias Gtfs.Trip
+
+  @type t :: %{Trip.id() => [StopTimeUpdate.t()]}
+
+  @spec stop_time_updates_for_trip(t(), Trip.id()) :: [StopTimeUpdate.t()]
+  def stop_time_updates_for_trip(stop_time_updates_by_trip, trip_id) do
+    Map.get(stop_time_updates_by_trip, trip_id, [])
+  end
+
+  @spec trip_ids(t()) :: [Trip.id()]
+  def trip_ids(stop_time_updates_by_trip) do
+    Map.keys(stop_time_updates_by_trip)
```

Comment:
This `trip_ids()` function is only used in `stop_time_updates.ex`, and the `stop_time_updates_for_trips()` function is only used in `block_waiver.ex`. What do you think about moving these functions to those files, or inlining them since they're simple `Map` calls, so this module can go away?

Reply:
To me this is a collection of data (kind of a DTO) that we have during a part of our flow and I find it helpful to name it and encapsulate actions on it. Yes it happens to use a map and live in memory right now, but other objects that interact with it don't need to worry about those details.